docs/concepts/getting_started/adding_js_and_css.md

```diff
@@ -177,25 +177,103 @@ So in your HTML, you may see something like this:
 
 Finally, we return to our Python component in `calendar.py` to tie this together.
 
-To link JS and CSS defined in other files, use the `Media` nested class
-([Learn more about using Media](../fundamentals/defining_js_css_html_files.md)).
+To link JS and CSS defined in other files, use [`js_file`](../../../reference/api#django_components.Component.js_file)
+and [`css_file`](../../../reference/api#django_components.Component.css_file) attributes:
 
 ```python title="[project root]/components/calendar/calendar.py"
 from django_components import Component
 
 class Calendar(Component):
     template_name = "calendar.html"
+    js_file = "calendar.js"   # <--- new
+    css_file = "calendar.css"   # <--- new
+
+    def get_context_data(self):
+        return {
+            "date": "1970-01-01",
+        }
+```
+
+And that's it! If you were to embed this component in an HTML, django-components will
+automatically embed the associated JS and CSS.
+
+!!! note
+
+    Similarly to the template file, the JS and CSS file paths can be either:
+
+    1. Relative to the Python component file (as seen above),
+    2. Relative to any of the component directories as defined by
+    [`COMPONENTS.dirs`](../../reference/settings.md#django_components.app_settings.ComponentsSettings.dirs)
+    and/or [`COMPONENTS.app_dirs`](../../reference/settings.md#django_components.app_settings.ComponentsSettings.app_dirs)
+    (e.g. `[your apps]/components` dir and `[project root]/components`)
+    3. Relative to any of the directories defined by `STATICFILES_DIRS`.
+
+
+<!-- TODO: UPDATE AFTER AT LEAST ONE IMPLEMENTED
+!!! info
+
+    Special role of `css` and `js`:
+
+    The "primary" JS and CSS you that specify via `js/css` and `js/css_file` have special role in multitudes of django-components'
+    features:
+    - CSS scoping [a la Vue](https://vuejs.org/api/sfc-css-features.html#scoped-css)
+    - CSS variables from Python are available
+    - JS variables from Python are available
+    - JS can pass a callback to special JS method `$onLoad()`, which will be called every time
+      a component is rendered on the page.
+
+    This is not true for JS and CSS defined in `Media.js/css`, where the linked JS / CSS are rendered as is.
+-->
+
+### 5. Link additional JS and CSS to a component
```

Comment:
Added this section to the tutorial which now describes the files linked to with `Media.js/css` as "additional JS/CSS"